core/modes/diff_mode.go

```diff
@@ -74,9 +74,10 @@ func (this *DiffMode) Process(request *http.Request, details models.RequestDetai
 		respHeaders := util.GetResponseHeaders(actualResponse)
 
 		actualResponseDetails := &models.ResponseDetails{
-			Status:  actualResponse.StatusCode,
-			Body:    respBody,
-			Headers: respHeaders,
+			Status:     actualResponse.StatusCode,
+			Body:       respBody,
+			Headers:    respHeaders,
+			FixedDelay: int(duration.Milliseconds()),
```

Comment:
I don't think we should record it for diff mode, we don't want to fail the diffing because of variation in the delay value.

Reply:
done